frontend/awx/main/routes/useAwxManagementJobsRoutes.tsx

```diff
@@ -55,7 +30,7 @@ export function useAwxManagementJobsRoutes() {
             {
               id: AwxRoute.ManagementJobSchedules,
               path: 'schedules',
-              element: <Schedules sublistEndpoint={awxAPI`/system_job_templates`} />,
+              element: <ManagementJobSchedules />,
```

Comment:
Do you want to remove this for route along with `ManagementJobSchedules.tsx` for now and then add it back when making a PR for [AAP-8220](https://issues.redhat.com/browse/AAP-8220 )? Was thinking that way every PR tracks the exact work for a specific task.
```suggestion
              element: <PageNotImplemented />,
```